components/automate-cluster-ctl/libexec/cluster-info

```diff
@@ -40,7 +40,7 @@ class AutomateClusterInfo < AutomateCluster::Command
                       multiline: true,
                       padding: [0,0,0,0],
                       alignments: [:right, :left],
-                      column_widths: 200,
+                      width: 150,
```

Comment:
Can we have a screen shot of the output of info command, to know how it looks? It will give final output prospective as per this column and width changes. Attach it in the PR.

Reply:
![Screenshot from 2022-01-27 15-43-04](https://user-images.githubusercontent.com/60912283/151338565-c19207e1-3746-4096-b173-09e2ce1df4c4.png)